kimchi/src/circuits/expr.rs

```diff
@@ -37,6 +56,8 @@ pub struct Constants<F> {
     pub endo_coefficient: F,
     /// The MDS matrix
     pub mds: Vec<Vec<F>>,
+    /// Foreign field modulus
+    pub foreign_modulus: Vec<F>,
```

Comment:
This appears to be unused.. Delete it until it's needed?

Reply:
Will convert this PR & branch into `RangeCheck` and move the foreign mul code into another branch.